### ISSUE TITLE
Release 0.2.2 appimage reports version as NVIM v0.2.3-1-gd6f9d1df0

### ISSUE THREAD

**Issue by MaxNoe** [neovim/neovim#8116]
### Steps to reproduce

Download the app image  from here:

```
$ curl -LO https://github.com/neovim/neovim/releases/download/v0.2.2/nvim.appimage
$ chmod +x nvim.appimage
$ ./nvim.appimage --version
NVIM v0.2.3-1-gd6f9d1df0
```

### Actual behaviour

Version is v0.2.3

### Expected behaviour

Version should be v0.2.2

**Comment by stsewd** [comment]
This is a duplicate of https://github.com/neovim/neovim/issues/7645

**Comment by MaxNoe** [comment]
The issue is closed despite this being still the case?

**Comment by jamessan** [comment]
The appimage was created from a single commit after the release of 0.2.2.  We can't go back and fix that and, as Justin mentioned in #7645, it will be addressed for 0.2.3.

There's nothing we can do for 0.2.2.  That's the way releases work.

**Comment by jamessan** [comment]
Either way, there were existing issues so there was no reason to create a new one.
